packages/vscode-extension/package.json

```diff
@@ -436,6 +436,16 @@
           ],
           "default": true,
           "description": "Enable linting errors for Cypher queries"
+        },
+        "neo4j.features.useVersionedLinters": {
```

Comment:
I needed to add this to the configuration because otherwise we cannot enable it from the tests